app/scss/questionPage.scss

```diff
@@ -16,6 +17,35 @@
     height: 100%;
 }
 
+.question-nav {
+    height: 60px;
+    margin: 20px 0 20px 0;
+    background-color: #ffffff;
+}
+
+.question-nav span {
+    position: relative;
+    width: 25px;
+    padding: 8px 0 32px 0;
+    display: inline-block;
+    color: black;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+.flag {
+    display: inline-block;
```

Comment:
Not sure if this does anything in the browser

Reply:
*when disabled in the browser

Reply:
fixed in 84011a6